apps/service_define/lib/define.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Define do
```

Comment:
I appreciate this decent module doc. Thank you.

Reply:
Wait, it's gone now!?

Reply:
I'm so sorry

Reply:
🔥